libp2p/connection.nim

```diff
@@ -113,7 +113,7 @@ method close*(s: Connection) {.async, gcsafe.} =
     s.isClosed = true
   trace "connection closed", closed = s.closed
 
-proc readLp*(s: Connection): Future[seq[byte]] {.async, gcsafe.} =
+proc readLp*(s: Connection; maxSize: int = DefaultReadSize): Future[seq[byte]] {.async, gcsafe.} =
```

Comment:
We don't need this, `readLp` means `read length prefixed`, there are other methods such as `read` or `readExactly` that allow you to specify the size. Modifying `readLp` to take a length defeats it's purpose.

Reply:
Hmm true
Alright, I will just keep the secio fixes and merge those, I will rewrite the test using `read`